spec/api_spec.rb

```diff
@@ -137,6 +137,13 @@
     expect(resource["quality_analysis"]).to have_key("focus")
   end
 
+  it "should support the cinemagraph_analysis parameter" do
+    resource = @api.resource(test_id_1, :cinemagraph_analysis => true)
```

Comment:
It would be better to use mock and check that `cinemagraph_analysis` is present in the request